wiki/Article_Styling_Criteria/en.md

```diff
@@ -349,59 +511,190 @@ All beatmapset difficulty URLs looks like this: `https://osu.ppy.sh/beatmapsets/
+
+You must set these settings before taking a screenshot of the osu! client (settings not stated below are assumed to be at their defaults):
+
+-   Select language: `English`
+-   Prefer metadata in original language: `Enabled`
+-   Release stream: `Stable`
+-   Frame limiter: `VSync`
+-   Detect performance issues: `Disabled`
+-   Resolution: `1280x720`
+-   Fullscreen mode: `Disabled`
+-   Screenshot format: `JPEG (web-friendly)`
+-   Parallax: `Disabled`
+-   Menu tips: `Disabled`
+-   Seasonal backgrounds: `Never`
+-   Show thumbnails: `Enabled`
+-   Background dim: `30%`
+-   Progress display: `Top-Right (Pie)`
+-   Score meter type: `Hit error`
+-   Score meter size: `1x`
+-   Always show key overlay: `Enabled`
+-   Current skin: `Default` (first option)
+
+#### Image links
+
+Never use image links! Image links are links with the image as the link text. On a PC platform, they will work. However, on a mobile platform, they will not work as the pointer event will trigger the hover text, not the link.
 
 ---
 
-It is suggested to add a subfolder for the images (e.g. `img/` for all and any images). Images that are used in multiple different articles must go to the `/wiki/shared/` folder while those used for a single article are to be placed inside their folders for linking.
+Flag icons next to userlinks must be separate from the username. For example:
+
+```
+Do not: [![][flag_AU] peppy](/users/2)
+
+Do:     ![][flag_AU] [peppy](/users/2)
+```
+
+#### Flag icons
+
+*For a list of flag icons, see: [issue \#328 (GitHub)](https://github.com/ppy/osu-wiki/issues/328)*
+
+All of the flag icons use the two letter code (in all capital letters) and ends with `.gif`. With a few exceptions being the multi-flag icons which are in the PNG format. When adding a flag inline, use this format:
+
+```
+![](/wiki/shared/flag/xx.gif)
+```
+
+`xx` is the [ISO 3166-2 (Wikipedia)](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) two-lettered country code for the flag.
+
+Adding the full country names in the alt text is optional.
 
 ### Tables
 
-Tables use the GFM syntax.
+Tables in the osu! wiki support headings along the first row. If you cannot create a table because something would not work without HTML (e.g. lists inside tables), you need to reconsider the layout you are trying to use.
 
-If you cannot create a table because _something_ would not work without HTML (e.g. lists inside tables), you need to reconsider the layout you are trying to use.
+Tables must never be beautified. They must have a vertical bar (`|`) on the left and right sides and the tect of each cell must be padded with one space on both sides. Empty cells must use a vertical bar (`|`) followed by two spaces then another vertical bar (`|`).
 
-Here is a tool to help visualize tables:
+The delimter row (the row that separates the table head from the table body) must use only three characters per column (and be padded with a space on both sides), which must look like one of the following:
```

Comment:
`delimter` -> `delimiter` (two instances)

Somehow my old review got outdated, and it hasn't been resolved.

Reply:
Oh, so that's what got dropped.